### DISCUSSION TITLE
Release 1.0.16

### DISCUSSION THREAD

**Discussion by moabu** [JanssenProject/jans#5528] [Polls]
https://github.com/JanssenProject/jans/milestone/19

**Comment by manojs1978** [comment]
https://github.com/JanssenProject/jans/issues/5647

**Comment by moabu** [comment]
https://github.com/JanssenProject/jans/issues/5693

**Comment by MohitMaliFtechiz** [comment]
#5693 server ip address should be change to server ip address when provided the value but it returns to 225.225.225.225 after sometime

**Reply by manojs1978** [reply]
https://github.com/JanssenProject/jans/issues/5695 
getting some sql exception in jans-link with jans mysql DB

**Comment by moabu** [comment]
https://github.com/JanssenProject/jans/releases/tag/v1.0.16
